book/SUMMARY.md

```diff
@@ -40,4 +40,4 @@
 
 - [Recommended Settings](./developers/recommended-settings.md)
 
-- [Building Groth16 Artifacts](./developers/building-groth16-artifacts.md)
```

Comment:
nit: Fix, name of `build-groth16-artifacts.md` should be changed to `building-plonk-artifacts`.